examples/filebrowser/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["briefcase"]
+
+[tool.briefcase]
+project_name = "Android Filebrowser Demo"
+bundle = "org.beeware"
+version = "0.3.0.dev25"
+url = "https://beeware.org"
+license = "BSD license"
+author = 'Tiberius Yak'
+author_email = "tiberius@beeware.org"
+
+[tool.briefcase.app.filebrowser]
+formal_name = "Android Filebrowser Demo"
+description = "A testing app"
+sources = ['filebrowser']
+requires = [
+    'c:/Projects/Python/Toga/src/core'
```

Comment:
These c:/ paths will have to be replaced with names that work on others' machines, e.g. `toga-core` here, I believe.

I definitely appreciate that it's finnicky to swap these in when you're developing toga, then swap them out to match the style of other examples' `pyproject.toml`.

Reply:
Yeah, you're right there. 
I just did not know, whether we will keep the filebrowser example or if we just use it until the PR is accepted.
There actually **is** already an example for dialogs, but I didn't want to change that completly (due to the async nature).
Besides, the existing dialog examle might have a too large height for Android where scroll view is not yet working.
Some controls might acutally be outside the visible area.